org.osgi.test.cases.cm/src/org/osgi/test/cases/cm/junit/CMCoordinationTestCase.java

```diff
@@ -436,8 +436,11 @@ public void updated(Dictionary<String, ? > properties)
 
 		// wait and verify listener
 		sleep();
-		assertEquals(1, events.size());
-		assertFalse(events.get(0));
+		assertTrue(events.size() <= 2);
```

Comment:
Why not **exactly** 2?

just for the record quoting the output of the [failed testcase](https://github.com/eclipse-equinox/equinox/pull/559/checks?check_run_id=24695693302):

```
expected:<1> but was:<2>
junit.framework.AssertionFailedError: expected:<1> but was:<2>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at junit.framework.Assert.assertEquals(Assert.java:241)
	at junit.framework.TestCase.assertEquals(TestCase.java:384)
	at org.osgi.test.cases.cm.junit.CMCoordinationTestCase.testCoordinatedConfigurationOnBeforeRegisteredManagedService(CMCoordinationTestCase.java:439)
```

Reply:
Then felix command will fail.

Reply:
As the spec now mandates this I updated the testcase accordingly.